db/migrate/20230203070356_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :users
```

Comment:
- [ ] Please remove this field as it is not part of the required fields for the user table